client/src/components/modals/pick-entity-or-conversion.vue

```diff
@@ -1,58 +1,55 @@
 <template>
-  <v-dialog v-model="showModal" :max-width="width" hide-overlay>
-    <v-card class="pa-0 no-border">
+  <v-dialog v-model="showModal" :max-width="width" persistent>
+    <v-card class="pa-6 no-border pick-entity-card">
+      <v-row class="pl-1 mx-0">
+        <v-col cols="11">
+          <label>{{ locationText }}:</label>
+        </v-col>
+        <v-col cols="1">
+          <v-icon md color="primary" @click="showModal = false">mdi-close</v-icon>
+        </v-col>
+      </v-row>
       <template v-if="isConversion">
-        <v-card-text class="py-4 text-center">
-          <v-container fluid>
+        <v-card-text>
+          <v-container fluid class="px-4">
             <v-row>
-              <v-col cols="12" class="h5">Possible Conversions</v-col>
+              <v-col cols="12" class="copy-bold">Alterations</v-col>
             </v-row>
             <v-row v-for="(conversion, i) in conversionTypes" :key="'conv-'+i">
               <v-col cols="12" class="clickable-cell"
                      :id="conversion.value"
                      @click="chooseConversion(conversion)">
-                <v-tooltip bottom max-width="500" open-delay="500">
+                <v-tooltip bottom content-class="bottom-tooltip">
                   <template v-slot:activator="scope">
-                    <button v-on="scope.on" class="link-sm-sans-ul text-left">{{ conversion.desc }}</button>
+                    <button v-on="scope.on" class="link-sm-sans-ul entity-link">{{ conversion.desc }}</button>
                   </template>
-                  <ul class="ma-0 pa-0">
-                    <li class="no-bullet"
-                        v-html="conversion.blurb"
-                        :key="i + '-blurb'" />
-                  </ul>
+                  <div v-for="(text, i) in conversion.blurb" :key="i + '-blurb'">
```

Comment:
I'm not sure whether keys that start with a number work...?

Can you just use `:key:"i"` here?

Reply:
We can for sure. Normally i'd be worried about duplicate keys but these blurbs are tool tips.

Reply:
Compromise: `blurb-x` ?

(It's the leading number that I'm uncomfortable with.)

Reply:
Definitely agree, 100% will do 👍